iati/tests/test_validator.py

```diff
@@ -349,6 +347,16 @@ def schema_incomplete_codelist(self):
         return schema
 
     @pytest.fixture
+    def schema_short_mapping_codelist(self):
+        """Return an Activity Schema with a Codelist that has a short `path` in the mapping file."""
+        schema = iati.core.default.activity_schema(None, False)
+        codelist = iati.core.default.codelists()['Language']
```

Comment:
Can access the `Language` code directly rather than accessing all of them first.